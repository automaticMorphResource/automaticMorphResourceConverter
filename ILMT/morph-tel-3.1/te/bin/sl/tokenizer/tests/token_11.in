ఆ చేపల రంగులు, వాటి శరీరంపై ఉండే మిలమిలలు, డిజైన్లను కళ్లారా చూడాల్సిందే తప్ప మాటల్లో మాత్రం వర్ణించేందుకు వీలుకాదు. 
