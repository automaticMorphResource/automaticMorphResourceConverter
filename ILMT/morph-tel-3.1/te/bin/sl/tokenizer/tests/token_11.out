<Sentence id="1">
1	ఆ	unk
2	చేపల	unk
3	రంగులు	unk
4	,	unk
5	వాటి	unk
6	శరీరంపై	unk
7	ఉండే	unk
8	మిలమిలలు	unk
9	,	unk
10	డిజైన్లను	unk
11	కళ్లారా	unk
12	చూడాల్సిందే	unk
13	తప్ప	unk
14	మాటల్లో	unk
15	మాత్రం	unk
16	వర్ణించేందుకు	unk
17	వీలుకాదు	unk
18	.	unk
</Sentence>

